__tests__/main.test.ts

```diff
@@ -155,11 +149,52 @@ describe("generate summaries", () => {
         },
       ],
     });
+    expect(annotations).not.toBeUndefined();
+    expect(annotations?.length).toEqual(1);
+    expect(annotations?.[0].start_line).toEqual(1);
+    expect(annotations?.[0].path).toEqual("./test.h");
+    expect(annotations?.[0].annotation_level).toEqual("failure");
+    expect(annotations?.[0].message).toEqual("FailFixture: failUnit failed.");
+  });
+  it("generates trimmed annotations", () => {
+    const results = main.generateAnnotationsFromResults({
+      results: [
+        {
+          passed: 1,
+          failures: 0,
+          ignored: 3,
+          failureDetails: [],
+          fixture: "test",
+        },
+        {
+          passed: 0,
+          failures: 1,
```

Comment:
Is this a plausible test, with `failures: 1` and `failureDetails` of length 2?

Reply:
Yes, the details can show different test failures for the fixture.